poly-synth/src/poly-synth.js

```diff
@@ -52,13 +55,17 @@ class PolySynth {
     this.minDetuneAmount = 0;
     this.maxDetuneAmount = 5;
 
+    this.minQ = 0;
+    this.maxQ = 100;
```

Comment:
The Q parameter for a lowpass filter is in dB.  Read the spec and choose different values. Or play around with http://googlechrome.github.io/web-audio-samples/samples/audio/mag-phase.html to see what makes sense. (There's a bug in that it doesn't know that the Q for lowpass/highpass filters is in dB.  I need to fix that.)